2025-09-24 19:59:19,817 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
2025-09-24 19:59:25,166 ERROR: Exception on /edit_profile [GET] [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/wtforms/form.py", line 209, in __call__
    return type.__call__(cls, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2025-09-24 19:59:30,231 ERROR: Exception on /edit_profile [GET] [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/wtforms/form.py", line 209, in __call__
    return type.__call__(cls, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2025-09-24 19:59:45,160 ERROR: Exception on /edit_profile [GET] [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/wtforms/form.py", line 209, in __call__
    return type.__call__(cls, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2025-09-24 20:00:06,736 ERROR: Exception on /edit_profile [GET] [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py:875]
Traceback (most recent call last):
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "/Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/venv/lib/python3.13/site-packages/wtforms/form.py", line 209, in __call__
    return type.__call__(cls, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2025-09-24 20:01:06,370 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
2025-09-26 11:14:47,792 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
2025-09-26 11:15:01,401 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
2025-09-26 17:02:17,755 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
2025-09-27 11:11:18,660 INFO: Microblog startup [in /Users/trevornance/coding/web_stuff/flask_mega_tutorial-microblog/app/__init__.py:51]
